HerPublicWebsite/Constants.cs

```diff
@@ -4,5 +4,5 @@ public static class Constants
 {
     public const string FEEDBACK_URL_DEFAULT = "https://forms.office.com/e/GHshSHbVep";
     public const string SERVICE_NAME = "Check if you are eligible for an energy grant for a home with no gas boiler";
-    public const string SERVICE_URL = "TODO: BEISHER-504 Add production landing page here";
+    public const string SERVICE_URL = "www.gov.uk/apply-home-upgrade-grant";
```

Comment:
Should this have a protocol (i.e. `https://`)?

Reply:
Good point!